JSDemos/Demos/Popup/Overview/Vue/description.md

```diff
@@ -6,31 +6,31 @@ To see step-by-step instructions on how to get started with the DevExtreme Popup
 
-- Custom close button    
+- **Custom close button**    
 This demo shows how to add custom buttons to a Popup. One of these buttons uses an [onClick](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick) handler to set the `popupVisible` variable to `false`, causing the Popup to disappear. Refer to the next section (**Configure the Popup**) to learn how you can populate a popup with custom controls.
 
-- On outside click    
-Enable the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#hideOnOutsideClick) property to allow users hide the Popup by clicking outside the component.
+- **On outside click**    
+Enable the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#hideOnOutsideClick) property to allow users to hide the Popup by clicking outside the component.
 
 ## Configure the Popup
 
 The Popup inner area is divided into three parts:
 
-- Top toolbar  
+- **Top toolbar**  
 
-    - Predefined    
-    Set [showTitle](Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) to `true` and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to specify the caption. If you do not disable the [showCloseButton](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showCloseButton) property, the Close button appears.
+    - **Predefined**    
+    Set [showTitle](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) to `true` and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to specify the caption. If you do not disable the [showCloseButton](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showCloseButton) property, the Close button appears.
     
-    - Custom   
-    Add [toolbarItems](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/) markup and set each item's [toolbar](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#toolbar) property to `top`. If you want to display an item in the overflow menu, as shown in this demo, set the item's [locateInMenu](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#locateInMenu) to `always`.
+    - **Custom**   
+    Add [toolbarItems](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/) markup and set each item's [toolbar](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#toolbar) property to `top`. If you want to display an item in the overflow menu, as shown in this demo, set the item's [locateInMenu](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#locateInMenu) property to `always`. You can also assign `never` to this property to keep the item outside the overflow menu, or `auto` to hide the item in the menu if the Popup's width decreases.
```

Comment:
Make an edit here to the last sentence as I did to a similar sentence a few paragraphs above.